samples/nrf9160/aws_fota/src/main.c

```diff
@@ -367,8 +367,8 @@ static int client_id_get(char *id_buf)
 {
 #if !defined(CONFIG_CLOUD_CLIENT_ID)
 	enum at_cmd_state at_state;
-	char imei_buf[IMEI_LEN + 5];
-	int err = at_cmd_write("AT+CGSN", imei_buf, (IMEI_LEN + 5), &at_state);
+	char imei_buf[IMEI_LEN];
```

Comment:
Is this the buffer in question? AFAICS it hasn't gotten any bigger, but probably just my saturday evening brain messing with me.

Reply:
This is the buffer `id_buf`

Reply:
`client_id_buf` ?

Reply:
Yes, `client_id_buf` is the buffer that has gotten a bit bigger. So it's the length calculation for `client_id_buf` that has changed.

So the IMEI buffer needs to be 19 so that it can fit the AT cmd response so maybe I should have named it at_cmd_rsp_buf but in the function, we are specifically asking for the IMEI through the AT command which is the reasoning behind the naming. If this is confusing I can change the name.